tests/analyze.R

```diff
@@ -106,20 +105,45 @@ ProcessAll = function(ctx) {
                   proportion = rappor$proportion,
                   dist = "rappor")
 
-  # Fill in zeros for values missing in RAPPOR.  It makes the ggplot bar plot
-  # look better.
+  # Fill in zeros for values missing in RAPPOR and in actual data
+  # Helps with false positive detection and makes the ggplot look better
   fill <- setdiff(actual$string, rappor$strings)
+  fpfill <- setdiff(rappor$strings, actual$string)
   if (length(fill) > 0) {
     z <- data.frame(index = StringToInt(fill),
                     proportion = 0.0,
                     dist = "rappor")
+    r <- rbind(r, z)
+  }
+  if (length(fpfill) > 0) {
+    # More strings detected in RAPPOR than present in actual distr
+    # These strings must be reported as false positives in metrics$fp_values
+    z <- data.frame(index = StringToInt(fpfill),
+                    proportion = 0.0,
+                    dist = "actual")
+    a <- rbind(a, z)
   } else {
-    z <- data.frame()
+    fpfill = ""
```

Comment:
Done. I think my workflow was a little different before. It works if fpfill returns character(0) when length(fpfill) == 0.